src/components/Trade/TradeInput.tsx

```diff
@@ -249,6 +249,9 @@ export const TradeInput = () => {
 
   const hasValidSellAmount = bnOrZero(sellAmountCryptoPrecision).gt(0)
 
+  // TODO(woodenfurniture): update swappers to specify this as with protocol fees
+  const networkFeeRequiresBalance = activeSwapperName !== SwapperName.CowSwap
```

Comment:
moved out from inside `getErrorTranslationKey` so we can apply this logic to max quotes too.